### DISCUSSION TITLE
Multiple Lights?

### DISCUSSION THREAD

**Discussion by fgigon** [maca88/SmartBikeLights#22] [Q&A]
Hi, I just noticed that support for controlling multiple headlights/taillights at once was added a while ago, do I understand correctly that this only means you can use two headlights OR two taillights of the same model at the same time? By now I have a Garmin Varia Lights (UT800 and RTL515) as well as Bontrager Lights (ION Pro RT and ION 200 RT) which I use in different combinations. Is there a way to control them all? Not necessarily different settings for each light, but just a headlight/taillight configuration that would apply to each headlight/taillight? Or is the only work-around to control one headlight/taillight with the SmartBikeLights, the other lights via the built-in Garmin light network? Thanks for the support and best wishes for 2022 !

**Comment by maca88** [comment]
Hi, best wishes to you too!

> this only means you can use two headlights OR two taillights of the same model at the same time?

It is not bound to the model but rather to light modes. You can control ION Pro RT and ION 200 RT at the same time because both lights share the same light modes. UT800 on the other hand does not share the same light modes, so you would need to have a different configuration for it.

> Is there a way to control them all?

You could control all of them by installing an additional Smart Bike Lights data field (https://apps.garmin.com/sl-SI/apps/2e27525f-3847-4e49-89af-673970692df6) and configure UT800 + RTL515 on one data field and ION Pro RT + ION 200 RT on the other.

> but just a headlight/taillight configuration that would apply to each headlight/taillight?

In case you will use one headlight and one taillight at a time, you can create two configurations, one for ION Pro RT/200 RT + RTL515 and the other for UT800 + RTL515. Then before riding you would need to select the correct configuration by using Garmin Connect Mobile.

**Comment by fgigon** [comment]
Hi
thanks for your quick reply. I will try the different possibilities.
Selecting the correct configuration in Garmin Connect Mobile before each
ride (depending on which lights are being used) seems a bit a hassle. But
do i understand correctly that if I install two instances of the data field
in the same activity profile, depending on what lights are near the
computer one datafield will work as intended while the other will just
produce an error (without further impact) ? or even if I mix the lights
from the two datafield configurations, one light would be controlled by one
datafield and the other by the other datafield? That would probably be the
best and easiest solution. Will try if this works without problems...
Best regards!

Am Mo., 3. Jan. 2022 um 16:48 Uhr schrieb maca88 ***@***.***>:

**Reply by maca88** [reply]
> if I install two instances of the data field in the same activity profile, depending on what lights are near the computer one datafield will work as intended while the other will just produce an error (without further impact) ?

Yes, that is correct. One data field will work as expected where the other will display `Error 3`, because the configuration light modes do not match. In your case, by using two data fields, one configured for UT800+RTL515 and the other for ION Pro/200 RT + RTL515, no matter how you will mix the lights, one data field will always work where the other will show `Error 3`.

**Comment by fgigon** [comment]
I tried it, it only works partially. I installed the second instance of the
datafield and configured both datafields in different constellations (#1
for ION 200 RT/ION Flare RT and #2 for Varia UT800/Varia RTL515, or #1 for
ION 200 RT/Varia RTL515 and #2 for Varia UT800/Varia RTL515), as long as
only the two lights that are together in one configuaration are near my
Edge 1030 everything is fine and one datafield works the other shows Error
3, but as soon as i mix the lights across both datafields (e.g. Varia UT800
and  ION Flare RT) I have an Error 3 on *both *datafields. The Garmin Light
Network still works however, so I will probably just configure the two
combinations I use most with the 2 datafields, for the other 2 combinations
the Garmin Light Network will have to do the job...


Am Di., 4. Jan. 2022 um 19:24 Uhr schrieb maca88 ***@***.***>:

**Reply by maca88** [reply]
> as soon as i mix the lights across both datafields (e.g. Varia UT800 and  ION Flare RT) I have an Error 3 on *both *datafields

For UT800 + RTL515 you can fix this by setting the serial number for the UT800 in the Lights configuration. By setting the serial number, the data field will not try to pair a different light, so Error 3 will never occur. The same cannot be done for Ion Pro/200 RT, because currently the data field support setting only one serial number per light type (headlight/taillight).
